test/SlowTests/RavenDB-21679.cs

```diff
@@ -247,5 +248,59 @@ public async Task SampleDataWithDifferentCertificates(Options options)
                 Assert.Equal(0, indexDefinitions.Length);
             }
         }
+
+        [RavenTheory(RavenTestCategory.Certificates | RavenTestCategory.BackupExportImport)]
```

Comment:
Also add a test that shows that we throw an exception for a Read/Write certificate.

Reply:
Done